2048game/Mmove_2048.py

```diff
@@ -0,0 +1,90 @@
+import random
+
+def move_row_left(row) :
+    """Moves all tiles to the left according to the rules"""
+    taille = len(row)
+    hasBeenModified = [0] * taille 
+    """This list tells us if a tile has already merged during this move (and thus must not be moved again)"""
+    for position in range(taille) :
+        leftmostTileValue,leftmostTilePosition = row_check_tile_direction(row,"left",position) #On récupère la valeur de la première tuile rencontrée
+        if leftmostTileValue == row[position] and hasBeenModified[leftmostTilePosition] == 0 :
+            row[position] = " "
+            row[leftmostTilePosition] = str(2*int(leftmostTileValue)) #On fusionne les deux tuiles !
+            hasBeenModified[leftmostTilePosition] = 1
+        else :
+            tempValue = row[position]
+            row[position] = " "
+            row[leftmostTilePosition + 1] = tempValue
+    return row
+    
+def move_row_right(row) : 
```

Comment:
this function seems quite similar to the one above, could you find a way not to replicate code ?

Reply:
Mayhaps, but it would have taken more time.